cmd/gidari.go

```diff
@@ -69,8 +69,6 @@ func run(configFilepath string, verboseLogging bool, _ []string) {
 		log.Fatalf("error creating config: %v", err)
 	}
 
-	cfg.Logger = logrus.New()
```

Comment:
Good call!

Reply:
But I really think that it should be injected. Even if the signature is related to logrus. Did you check if logrus has a specific test object? I don't know this library.